WindowsRunnerCSharp/WindowsRunnerCSharp/WindowsRunnerCSharp.csproj

```diff
@@ -51,7 +51,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="com.playfab.csharpgsdk">
-      <Version>0.7.190313</Version>
+      <Version>0.8.190314</Version>
```

Comment:
Have you tested if uploading the zip file to one of the Thunderhead test titles gets the server to standingBy?

Reply:
I have not. That seems like a good thing to do, but I am not sure if I will get to it tomorrow unless you think it is higher priority than other investigations.

I did update the asset we use for golden path tests and that is working.